src/App.js

```diff
@@ -1,10 +1,32 @@
+import React from 'react';
 import Calculator from './components/Calculator';
 import './App.css';
+import Quotes from './components/quotes';
 
 function App() {
+  const [quote, setQuote] = React.useState(['loading...']);
+  const [result, setResult] = React.useState({ total: null, next: null, operation: null });
+
+  React.useEffect(() => {
+    setQuote('loading...');
+    fetch('https://api.api-ninjas.com/v1/quotes?category=computers', {
+      method: 'GET',
+      headers: {
+        'X-API-Key': 'OAcvP3F39QWUffY523tlRg==3Y15nTus6njhSQ1w',
+      },
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        const res = `${data[0].quote} - ${data[0].author}`;
+        setQuote(res);
+      })
+      .catch((error) => setQuote(['error ', error.message, ' quote']));
+  }, []);
```

Comment:
- [ ] Great job working on fetching the quotes from API but I think it would be great if you made the functionality in the Quote component then you import it here in app component since it is mentioed in the requirements 🔥🚀⭐ so  please update it ⭐🚀

![image](https://github.com/rachaouldbabaali/math-magicians/assets/58553711/80218397-8ddd-4dc6-b507-1988c2b31565)